### PR TITLE
Booksetup

### PR DESCRIPTION
**Pull request by vikas924**
Add two pages and use react routes

Display books

Add add functionality

Store in local storage

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+import Categories from './components/categories';
+import Books from './components/showbooks';
+
+function App() {
+  return (
+    <div className="wraper">
+      <Routes>
+        <Route element={<Navbar />}>
+          <Route index element={<Books />} />
+          <Route path="Categories" element={<Categories />} />
```
Comment (franclobo) [inline src/App.js]: - [x] Please 🙏🏽 correct your code for the better practice of React components, Navbar is not a route, all the route has path and elements attributes, all endpoints should be write in lowercase, e.g. 👇🏽 

```javascript
<div className="wraper">
<Navbar />
      <Routes>
          <Route path="/" element={<Books />} />
          <Route path="/categories" element={<Categories />} />
```

File: src/components/layout.jsx [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+        class: 'links hide',
+      }));
+    }
+  }
+
+  return (
+    <>
+      <header className="navbar">
+        <div className="navdiv">
+          <h1 className="header">Math Magicians</h1>
```
Comment (franclobo) [inline src/components/layout.jsx]: - [x] Is this Math Magicians or Bookstore project? Please 🙏🏽 use the correct title for this milestone.

File: src/components/showbooks.js [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+import Form from './form';
+
+export default function Books() {
+  function getInitialbooks() {
+    // getting stored items
+    const temp = localStorage.getItem('books');
+    const savedBooks = JSON.parse(temp);
+    return savedBooks || [
+      { id: 1, title: 'hello', author: 'noone' },
+      { id: 2, title: 'hel', author: 'one' },
```
Comment (franclobo) [inline src/components/showbooks.js]: - [x] What does "hel" mean? Please 🙏🏽 correct the word for professional documentation.

File: src/components/showbooks.js [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+import Book from './book';
+import Form from './form';
+
+export default function Books() {
+  function getInitialbooks() {
+    // getting stored items
+    const temp = localStorage.getItem('books');
+    const savedBooks = JSON.parse(temp);
+    return savedBooks || [
+      { id: 1, title: 'hello', author: 'noone' },
```
Comment (franclobo) [inline src/components/showbooks.js]: - [x] What does "noone" mean? Please 🙏🏽 correct the word for professional documentation.

**Review by franclobo — CHANGES_REQUESTED** [review]
Hi @vikas924, 

# Required Changes ♻️

so-close-jim-halpert

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- No linter errors ✔️ 
- Descriptive PR ✔️ 
- Correct Git flow ✔️ 
- Correct React initialize ✔️ 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [ ] Correct the logo in your README it should be Bookstore, not Math Magicians

image


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by tresorsawasawa — APPROVED** [review]
# STATUS: APPROVE :heavy_check_mark: :heavy_check_mark:  :champagne: 

Hi @vikas924 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 :clap: :clap: :100: 

nodding-yes

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Your README file is professional :heavy_check_mark: 

## Optional suggestions
_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**